DuckDuckGo/Fire/View/Fire.storyboard

```diff
@@ -29,7 +29,7 @@
                                         <rect key="frame" x="10" y="10" width="300" height="200"/>
                                         <userDefinedRuntimeAttributes>
                                             <userDefinedRuntimeAttribute type="color" keyPath="backgroundColor">
-                                                <color key="value" name="HomePageBackgroundColor"/>
+                                                <color key="value" name="NewTabPageBackgroundColor"/>
```

Comment:
This is a fix for the modal which would previously appear with white background at all times (HomePageBackgroundColor was deleted a few releases ago 🙈 guess by whom)